src/layers/conv.jl

```diff
@@ -506,3 +506,44 @@ function Base.show(io::IO, l::EdgeConv)
     print(io, ", aggr=", AGGR2STR[l.aggr])
     print(io, ")")
 end
+
+
+"""
+    GINConv([graph, ] nn, eps, train_eps
+"""
+
+struct GINConv{V<:AbstractFeaturedGraph,R<:Real} <: MessagePassing
+    fg::V
+    nn
+    eps::R
+    train_eps::Bool
+end
+
+function GINConv(adj::AbstractMatrix, nn, eps, train_eps)
+    fg = FeaturedGraph(adj)
+    GINConv(fg, nn, eps, train_eps)
+end
+
+function GINConv(nn, eps, train_eps)
```

Comment:
You can get some type `T` from `nn` and infers `eps=zero(T)` for default value.

Reply:
Done.